.github/workflows/release.yml

```diff
@@ -275,19 +275,19 @@ jobs:
 
           # Remove backslashes before colons in commit details
           new_fix_notes=$(echo "${new_fix_notes}" | sed 's/\\:/:/g')
-          
```

Comment:
- The variable RELEASE_NOTES is being used inappropriately in the updated_content assignment.